tools/applicationinsights-web-snippet/src/applicationinsights-web-snippet.ts

```diff
@@ -10,20 +12,17 @@ function webSnippetVersion() {
     return "";
 }
 
-function substituteInstrumentationKey(key: string) {
-    let userSnippet: String = webSnippet;
-    if (key && key.trim() !== "") {
-        userSnippet = webSnippet.replace("InstrumentationKey=INSTRUMENTATION_KEY", key);
+function getSdkLoaderScript(config: SdkLoaderConfig) {
+    let snippet: string = webSnippet;
+    if (config && config.connectionString) {
+        snippet = webSnippetCs.replace("YOUR_CONNECTION_STRING", config.connectionString);
+    } else if (config && config.instrumentationKey) {
+        snippet = webSnippet.replace("InstrumentationKey=INSTRUMENTATION_KEY", config.instrumentationKey);
+    } else {
+        console.log("No instrumentationKey or connectionString was provided to the init function");
```

Comment:
We should not be using `console.log`, which is a little hard with this component as we don't have any other way to get "error" information out.

Additionally, this will often not running on a browser... So there will be no console. So in this specific case I think we should just return the default snippet..